prettier/examples/prettier-format-from-gitrepo.yaml

```diff
@@ -0,0 +1,54 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: workspace-prettier-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 500Mi
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: prettier-pipeline-check
+spec:
+  workspaces:
+  - name: shared-workspace
+    # description: The git repo will be cloned into this workspace and the config-defaults.yaml will be read from it.
+  tasks:
+  - name: fetch-repo
+    taskRef:
+      name: git-clone
+    workspaces:
+    - name: output
+      workspace: shared-workspace
+    params:
+    - name: url
+      value: https://github.com/tektoncd/pipeline.git
```

Comment:
Let's pin the revision fetched to the SHA of this commit. We don't want this test breaking without an explicit change.

Reply:
I'm not sure what you mean. Does this mean that this way would probably lead to this test breaking?

Reply:
Talked about this more in chat. Intent here is to pin the SHA being used so that an unexpected change to master doesn't cause this test to suddenly fail.